module/regenerator.py

```diff
@@ -138,16 +147,29 @@ def load_from_scheduler(self, sched):
         self.contactgroups = c.contactgroups
         self.timeperiods = c.timeperiods
         self.commands = c.commands
+        # WebUI - Manage notification ways
         self.notificationways = c.notificationways
+
         # We also load the realm
         for h in self.hosts:
+            # WebUI - Manage realms if declared (Alignak)
             if getattr(h, 'realm_name', None):
                 self.realms.add(h.realm_name)
```

Comment:
In Shinken's regenerator, there is a break after this line. I don't know why, but no break is a behaviour change.

Reply:
OK, the break is done later, sorry.